### DISCUSSION TITLE
Creating local shortcuts for opening a Trilium note?

### DISCUSSION THREAD

**Discussion by IssbesAstieris** [TriliumNext/Trilium#3140] [Ideas] [answered]
Hello to Zadam and all fellow Trilium users, 
   I apologize in advance if the idea was already brought up or sounds silly, it crept upon me right in the middle of work. 
   @zadam, I've been aware for some time that [opening local files from within Trilium (using a link pointing toward said-files) was requested](https://github.com/zadam/trilium/discussions/1549), although not implemented. What about the other way around: for a local file to act as a shortcut pointing toward a given Trilium note? I was thinking about something similar to the OS-native shortcut files, but one that would require e.g. Windows to associate trilium.exe as the chosen program for opening a ".something" file acting as such. 
   Theoretically speaking, do you think it's possible to implement that kind of feature?

**Comment by XXXJetfireXXX** [comment]
Not sure about the relevance of such feature and if it's implementable, but there's an alternative I can suggest:
![image](https://user-images.githubusercontent.com/43646178/190853733-2a777b80-bda1-430f-8f40-12e7515dfc08.png)
By adding "bookmarked" tag you can add a bookmark to the left panel, leading to the note you want.
![image](https://user-images.githubusercontent.com/43646178/190853765-1f516ce7-9331-4f34-a7dd-957d9a1c74a8.png)
I hope this helps :)

**Reply by XXXJetfireXXX** [reply]
And if you want to store lots of bookmarks, you can make a bookmark folder by using #bookmarkFolder attribute as well.
More information can be found here: https://github.com/zadam/trilium/wiki/Bookmarks

**Reply by IssbesAstieris** [reply]
Hello and thanks for your answer @XXXJetfireXXX ! 
I've been using the #bookmarked attribute for a while; today, thanks to you, I discovered the #bookmarkFolder attribute... which I'm ashamed to admit I didn't know of until now, so I'm compelled to reread the entire wiki! 😅
I agree it should work very well for quick access to a lot of notes from within Trilium, although it's less situational than what I had in mind (i.e. placing a shortcut in strategic places within a PC's folders, so that one could instantly open a relevant note while browsing the file explorer). Still, I'm very grateful for your suggestion (now to facelift my bookmarks)!

**Comment by zadam** [comment, marked as answer]
Hi, this is something I'd very like to have as well, but IIRC it's not easy to do across all supported platforms.

Also discussed here: https://github.com/zadam/trilium/issues/1012

**Reply by IssbesAstieris** [reply]
Hello @zadam , 
My apologies for the duplicate; I've tried all sorts of wording beforehand yet didn't find this topic (courtesy of being a non-native English speaker, I suppose)...
I've read through it, so the situation is clearer to me now. I also see that you've just made a backlog out of it. Thank you very much for taking the time to answer; and more generally, thanks for your hard work on Trilium (which changed my student life for the best in barely a couple months). I'll be looking forward to future updates!
